server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ChunkOutputStream.java

```diff
@@ -60,15 +72,16 @@
        this.parentResponse = Objects.requireNonNull(parentResponse);
        this.completionSink = Objects.requireNonNull(completionSink);
        Objects.requireNonNull(reactorNettyResponse);
+
        this.byteSinkSupplier = () -> {
-           final Sinks.Many<Tuple2<byte[], CompletableFuture<Void>>> outSink = Sinks.many().multicast().onBackpressureBuffer();
-           final Flux<byte[]> byteFlux = outSink.asFlux().map(tup -> {
-                   tup.getT2().complete(null);
-                   return tup.getT1();
-               });
+           final Sinks.Many<Tuple2<byte[], CompletableFuture<Void>>> outSink =
+               Sinks.many().unicast().onBackpressureBuffer();
```

Comment:
Probably oversight.  @lalifk Do you recall?

Reply:
If my memory is correct, I chose `multicast()` mainly because I read in the docs that `unicast()` sinks can only have one subscriber. At the time I doubt I paused long enough to question whether that's good enough for our needs. I was driven by not bringing in that limitation. 
There are also other differences around whether or not data is buffered before subscription. I can't recall how I factored that into my decision.